src/syft/core/store/storeable_object.py

```diff
@@ -200,6 +203,8 @@ def pprint(self) -> str:
         output = f"{self.icon} ({self.class_name}) ("
         if hasattr(self.data, "pprint"):
             output += self.data.pprint  # type: ignore
+        elif self.data is not None:
```

Comment:
MAYBE: Remove this since we have the line above?

Reply:
They aren't the same thing. The line above only checks for the pprint property. There are a lot of Tensor types that dont have any pretty print methods yet, so they dont print anything.